lnd.go

```diff
@@ -237,7 +237,7 @@ func lndMain() error {
 		net.JoinHostPort("", strconv.Itoa(cfg.PeerPort)),
 	}
 	server, err := newServer(defaultListenAddrs, chanDB, activeChainControl,
-		idPrivKey)
+		idPrivKey, cfg)
```

Comment:
The config is currently a global, so need need to pass in.

Reply:
@bcongdon Was this eventually addressed?